easybuild/easyconfigs/s/STAR-Fusion/STAR-Fusion-1.5.0-foss-2018b-Perl-5.28.0.eb

```diff
@@ -16,14 +16,15 @@ description = """STAR-Fusion uses the STAR aligner to identify candidate fusion
  to map junction reads and spanning reads to a reference annotation set."""
 
 toolchain = {'name': 'foss', 'version': '2018b'}
+
 source_urls = ['https://github.com/%(name)s/%(name)s/releases/download/%(name)s-v%(version)s/']
 sources = ['%(name)s-v%(version)s.FULL.tar.gz']
 checksums = ['229cbf7980496d72de0bfab5bde7148ff3111a99b72c6169d1a04120a52508b4']
 
 dependencies = [
     ('Perl', '5.28.0'),
     ('DB', '18.1.25'),
-    ('STAR', '2.7.0d', '', ('foss', '2018b')),
+    ('STAR', '2.6.1c', '', ('foss', '2018b')),
```

Comment:
there is no need to add the toolchain here, so please replace with `('STAR', '2.6.1c')`